intake/interface/source/defined_plots.py

```diff
@@ -94,7 +123,9 @@ def source(self, source):
         BaseView.source.fset(self, source)
         if self.select:
             self.select.options = self.options
-        if source and dfviz and source.container == 'dataframe':
+        if source and hvplot and source.container == 'dataframe':
```

Comment:
Have you tried xarrays/arrays? Should ask upstream if that is supported (yet).

Reply:
We're working on xarray support so it's not supported yet.